tests/posix/common/src/clock.c

```diff
@@ -40,6 +42,12 @@ ZTEST(posix_apis, test_clock)
 	zassert_equal(secs_elapsed, SLEEP_SECONDS,
 			"POSIX clock API test failed");
 
+	ret = clock_getcpuclockid(0, &clock_id);
```

Comment:
I would suggest adding these to a separate ZTEST.

This file might get a bit of an update soon based on #67044

Reply:
Created separate ZTEST for clock_getcpuclockid